src/main/kotlin/com/atlassian/performance/tools/jiraactions/page/SingleSelect.kt

```diff
@@ -21,12 +21,12 @@ internal class SingleSelect(
             timeout = Duration.ofSeconds(8),
             condition = elementToBeClickable(input)
         )
-        inputElement.click()
+        inputElement.clear()
         // this is a workaround for the fact that clear() can be processed asynchronously and can remove
         // some of the keys sent to the browser. We have tested that under stress load, sending one backspace
         // before the value fixes this issue.
         // Whether we lose the backspace or not, it doesn't matter for the overall result.
-        inputElement.clear()
+        inputElement.click()
```

Comment:
Is this a consequence of the Selenium update?
If so, how did the behavior change?

Reply:
1. Yes.
2. I don't know. I hope it's not a timing change (but it could be). But in general it makes sense to clear it without clicking it.